src/styles/styles.js

```diff
@@ -2346,6 +2346,13 @@ const styles = {
         backgroundColor: colors.black,
         flex: 1,
     },
+
+    textWithEllipsis: {
+        textOverflow: 'ellipsis',
+        overflow: 'hidden',
+        ...whiteSpace.noWrap,
+        ...display.dInline,
```

Comment:
It will break Native.

Reply:
yeah I am aware. I am splitting it into two components.

Reply:
Fixed